### PR TITLE
Make project and topic page look good

### PR DESCRIPTION
**Pull request by Sbonelo01**
Related issues: https://umuzi-projects.monday.com/boards/1274815998/pulses/1437816996

Note: https://umuzi-projects.monday.com/boards/1274815998/pulses/1437816996#:~:text=Try%20not%20to,with%20each%20other.

## Screenshots/videos

details-page

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have merged the develop branch into my branch and fixed any merge conflicts
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have tested new or existing tests and made sure that they pass

### CODE REVIEW

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Think it looks good :)

Things to look at:
- The user avatars don't have tooltips anymore
- The cursor changes to a pointer when hovering section's body even though it's not clickable. That looks strange and misleading to me.
- Look out for long project details on mobile screens also

File: backend/frontend/templatetags/custom_tags.py [hunk 1]
```diff
@@ -10,7 +10,11 @@ def user_avatar(user, size="default"):
         "default": "w-[30px] h-[30px]",
         "small": "w-[15px] h-[15px] text-xs",
     }
-    initial = user.email[0].upper()
+    if hasattr(user, "email"):
+        initial = user.email[0].upper()
+    else:
+        initial =  user[0].upper()
+
```
Comment (Ngoakor12) [inline backend/frontend/templatetags/custom_tags.py:17]: Do we still need to do this? We can access the user instances to pass on the template just like we talked about on Slack or how it currently works in develop

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
There's still some overlay of elements on smaller screens:
image


Perhaps leave the icon without the "View more/less" text?

**Comment by Sbonelo01** [comment]
> There's still some overlay of elements on smaller screens: image
> 
> Perhaps leave the icon without the "View more/less" text?
- Desktop:
desktop-details-page
- Tablet:
tablet-details-page
- Mobile:
mobile-details-page

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Previous changes addressed.

Unfortunately, I picked up some more issues:

image
- Adding both the email and avatar for assignees messes with the alignment of the left side and right side. E.g. what is None for?

Also something I just realised, maybe you disagree:
image

- Does the content button or link need to be at the bottom where we have to scroll all the way down? Can it not fit somewhere higher up?

image

- It looks like the "Reviews" section heading is missing some padding, top and left

- Also not sure about the inline alignment of the reviewer and their reviews. Making the reviewer name as prominent as the actual review when we are looking at a list of reviews seems off, hierarchy-wise :shrug:

**Comment by Sbonelo01** [comment]
- Adding both the email and avatar for assignees messes with the alignment of the left side and right side

> That was pushed by mistake. Thanks for pointing it out

- Does the content button or link need to be at the bottom where we have to scroll all the way down? Can it not fit somewhere higher up?

> I see your point. But I think placing the buttons there might encourage learners to not look at repo details section. Ideally learners should also look at the PR reviews before making a Tilde review.

- It looks like the "Reviews" section heading is missing some padding, top and left

> Resolved

- Also not sure about the inline alignment of the reviewer and their reviews. Making the reviewer name as prominent as the actual review when we are looking at a list of reviews seems off, hierarchy-wise

> I agree. Resolved.

@Ngoakor12

**Review by Ngoakor12 — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**